Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -57,12 +57,9 @@ if (Platform.OS === 'android') {
   AndroidScrollView = ScrollViewNativeComponent;
   AndroidHorizontalScrollView = AndroidHorizontalScrollViewNativeComponent;
   AndroidHorizontalScrollContentView = AndroidHorizontalScrollContentViewNativeComponent;
-} else if (Platform.OS === 'ios') {
+} else {
```

Comment:
To maintain minimal delta's from upstream, I'd prefer this line to become `if (Platform.OS === 'ios' || Platform.OS === 'macos' /*TODO(macOS ISS#2323203)*/) {` and remove the surrounding changes.   Also, its possible the final `else` case is actually used by another platform, maybe react-native-web ?

Reply:
Good catch, I was looking at a different upstream tree.